kvdb-web/src/lib.rs

```diff
@@ -66,8 +68,11 @@ impl Database {
 				Err(err) => return future::Either::Right(future::err(err)),
 			};
 
-			// if we need more column than the latest version has,
-			// then bump the version (+ 1 for the default column)
+			// If we need more column than the latest version has,
+			// then bump the version (+ 1 for the default column).
+			// In order to bump the version, we close the database
+			// and reopen it with higher version that it was opened previously.
```

Comment:
"and reopen it with a higher version than it was opened with previously."

(gh suggestions don't work when reviewing new changes... odd!)

Reply:
```suggestion
			// and reopen it with a higher version than it was opened with previously.
```